### PR TITLE
Add member and change office address

### PR DESCRIPTION
**Pull request by trungking**

### CODE REVIEW

**Comment by trungking** [comment]
Preview: https://trungking.github.io/verichains.github.io/

**Comment by ldt116** [comment]
Tin's bio needs an update.

**Comment by ldt116** [comment]
> Preview: https://trungking.github.io/verichains.github.io/

Btw, we have a Private CI Runner, we can write a script that hosts a live preview for each PR. And when PR closed, CI will delete that preview automatically. That is better.

File: CNAME [hunk 1]
```diff
@@ -1 +0,0 @@
-www.verichains.io
```
Comment (ldt116) [inline CNAME:1]: Check this

**Pull request merged**